sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2643,11 +2646,77 @@ def setup_docker_user(self, cluster_config_file: str):
                                                     cluster_config_file)
         self.docker_user = docker_user
 
+    def _get_skylet_ssh_tunnel(self) -> Optional[SSHTunnelInfo]:
+        metadata = global_user_state.get_cluster_skylet_ssh_tunnel_metadata(
+            self.cluster_name)
+        if metadata is None:
+            return None
+        return SSHTunnelInfo(port=metadata[0], pid=metadata[1])
+
+    def _set_skylet_ssh_tunnel(self, tunnel: Optional[SSHTunnelInfo]) -> None:
+        if tunnel is None:
+            global_user_state.set_cluster_skylet_ssh_tunnel_metadata(
+                self.cluster_name, None)
+        else:
+            global_user_state.set_cluster_skylet_ssh_tunnel_metadata(
+                self.cluster_name, (tunnel.port, tunnel.pid))
```

Comment:
```suggestion
        global_user_state.set_cluster_skylet_ssh_tunnel_metadata(self.cluster_name, (tunnel.port, tunnel.pid) if tunnel is not None else None)
```
might be more compact